components/pages/Messages/Messages.module.css

```diff
@@ -253,3 +257,100 @@
   width: 100%;
   align-items: center;
 }
+.likeIconContainer {
+  width: max-content;
+  padding: 6px;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
If you make the `svg` have `display: block`, you won't need these.

Reply:
Yea, we changed the SVG to an emoji, and its display is block.